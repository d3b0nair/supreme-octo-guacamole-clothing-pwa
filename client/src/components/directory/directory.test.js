import React from "react";
import { render, cleanup } from "@testing-library/react";
import { Directory } from "./directory.component";

const component = (mockProps) => <Directory sections={[]} />;
afterEach(cleanup);

it("should mount", () => {
  const { container } = render(component());
  expect(container).toMatchSnapshot();
});
