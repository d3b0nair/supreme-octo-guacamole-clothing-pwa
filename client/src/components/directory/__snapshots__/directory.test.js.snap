// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should mount 1`] = `
<div>
  <div
    class="sc-jRQBWg kRidlN"
  />
</div>
`;
